dragonfly/player/player.go

```diff
@@ -256,6 +256,16 @@ func (p *Player) RemoveBossBar() {
 	p.session().RemoveBossBar()
 }
 
+// SetJoinMessage changes the message that appears when the player joins the server
+func (p *Player) SetJoinMessage(message string) {
+	p.s.SetJoinMessage(message)
+}
+
+// SetLeaveMessage changes the message that appears when the player leaves the server
+func (p *Player) SetLeaveMessage(message string) {
+	p.s.SetLeaveMessage(message)
+}
```

Comment:
I'm not too fond of these messages and the one in Session. Maybe just SetJoinMessage and SetLeaveMessage in server.go would be better?